themes/gfsc/layouts/index.html.html

```diff
@@ -24,6 +24,49 @@ <h3>{{$vennData.Params.center}}</h3>
 	 </div>
 {{ end }}
 
+
+<hr class="fancy fancy--no-phone" />
+{{ with $homepage.Resources.GetMatch "themes.md" }}
+  {{ $themes := $.Site.GetPage "/our-work/theme/" }}
+  get projects, count projects with category
+
+  <div class="homepage__themes">
+    <h2 class="homepage__themes__title">{{ .Title }}</h2>
+    <div class="homepage__themes__content">{{ .Content }}</div>
+    <ul class="windows windows--themes windows--orange">
+      {{ $projects := (where .Site.RegularPages "Section" "project") }}
+
+      {{ range $themes.RegularPages }}
+         {{ $theme := . }}
+         {{ $project_count := 0 }}
+
+         {{ range $projects }}
+            {{ $project_themes := (split .Params.themes " ") }}
+
+            {{ if (in $project_themes $theme.Params.key) }}
```

Comment:
`in`  works on strings so no need to split into an array but also it's not a problem for me

Reply:
i thought about that but discounted it as we may have themes that are sub-strings of other themes

Reply:
Good idea